pennylane/templates/subroutines/select.py

```diff
@@ -15,18 +15,28 @@
 Contains the Select template.
 """
 
-
 import copy
 import itertools
+from collections import Counter
 
 import pennylane as qml
 from pennylane import math
+from pennylane.decomposition import (
+    add_decomps,
+    adjoint_resource_rep,
+    controlled_resource_rep,
+    register_resources,
+)
+from pennylane.decomposition.resources import resource_rep
 from pennylane.operation import Operation
+from pennylane.ops import CNOT, X, adjoint, ctrl
+from pennylane.templates.subroutines.temporary_and import TemporaryAND
+from pennylane.wires import Wires
 
 
 class Select(Operation):
-    r"""Applies different operations depending on the state of
-    designated control qubits.
+    r"""The ``Select`` operator, also known as the **multiplexer**, or a multiplexed operation,
+    applies different operations depending on the state of designated control wires.
```

Comment:
```suggestion
    applies a set of operations depending on the state of designated control wires.
```

Reply:
The new version sounds a little like `ctrl(set_of_ops, control)`. :thinking: